src/main/java/uk/gov/hmcts/reform/civil/callback/CaseEvent.java

```diff
@@ -40,7 +40,8 @@ public enum CaseEvent {
     INITIATE_GENERAL_APPLICATION(USER),
     CREATE_SDO(USER),
     STANDARD_DIRECTION_ORDER_DJ(USER),
-
+    NotSuitable_SDO(USER),
```

Comment:
Why is it in lower case?

Reply:
I am merely following how it was named in PR CIV-3328 - I had to merge their changes into my branch to make the RPA for their unhappy path work.